src/core/dyad_core.c

```diff
@@ -30,15 +31,28 @@ const struct dyad_ctx dyad_ctx_default = {
     NULL    // cons_managed_path
 };
 
+struct dyad_kvs_response {
+    char* fpath;
+    uint32_t owner_rank;
+};
+typedef struct dyad_kvs_response dyad_kvs_response_t;
+
 static int gen_path_key (const char* str,
                          char* path_key,
                          const size_t len,
                          const uint32_t depth,
                          const uint32_t width)
 {
-    static const uint32_t seeds[10] = {104677u, 104681u, 104683u, 104693u,
-                                       104701u, 104707u, 104711u, 104717u,
-                                       104723u, 104729u};
+    static const uint32_t seeds[10] = {104677u,
```

Comment:
These individual values are not that meaningful, and you almost need not seem them. No need to make this change. Is this from clang format?

Reply:
Yes